checkstyle-tester/README.md

```diff
@@ -62,15 +65,15 @@ Please use `-Dcheckstyle.consoleOutput=true` option if you need to see all Check
 groovy launch.groovy projects-to-test-on.properties my_check.xml -Dcheckstyle.consoleOutput=true
 ```
 
-**Attention:** this project by deafult uses released version of Checkstyle and sevntu.checkstyle
-If you you need to use custom (snapshot) versions please update pom.xml to reference that versions ([checkstyle version](https://github.com/checkstyle/contribution/blob/master/checkstyle-tester/pom.xml#L29), [sevntu.checkstyle version](https://github.com/checkstyle/contribution/blob/master/checkstyle-tester/pom.xml#L34)), and please make sure that custom(newly generated) versions are located in your local maven repo 
+**Attention:** this project by deafult uses released version of [Checkstyle](https://github.com/checkstyle/checkstyle/releases/latest) and [sevntu.checkstyle](https://github.com/sevntu-checkstyle/sevntu.checkstyle/releases/latest).
+If you you need to use custom (snapshot) versions please update pom.xml to reference that versions ([checkstyle version](https://github.com/checkstyle/contribution/blob/master/checkstyle-tester/pom.xml#L15), [sevntu.checkstyle version](https://github.com/checkstyle/contribution/blob/master/checkstyle-tester/pom.xml#L16)), and please make sure that custom(newly generated) versions are located in your local maven repo 
```

Comment:
Let's avoid master links otherwise we will run into this problem again.
How about either, a specific commit or a search URL.

Ex:
https://github.com/checkstyle/contribution/blob/35d35dfcc48e2022403231e41aac8bf96126acc9/checkstyle-tester/pom.xml#L15
https://github.com/checkstyle/contribution/search?utf8=%E2%9C%93&q=path%3Acheckstyle-tester+filename%3Apom.xml+%22checkstyle.version%22&type=